config.parachain.test.yml

```diff
@@ -30,7 +39,8 @@ parachains:
     base: dev-local # the chain to use
     chainType: Development
     collators: # override collators
-      - ferdie
+      - alice # this imply //Alice
```

Comment:
Could I ask why we want to use two collators, alice and bob here?

Reply:
Shouldn't there be always two?! :thinking: but yeah, you are right, I don't want to be a ferdie-racist :blush: